lib/src/service_module/trademark_module/trademark_screen/layout/widgets/all_expandable_data.dart

```diff
@@ -3,139 +3,219 @@
-          padding:
-              EdgeInsets.only(left: 16.h, right: 16.h, top: 10.h, bottom: 10.h),
-        ),
-        ExpandableCards(
-          price: variables.staticData?.aed_10909_5Vat ?? '',
-          name: variables.staticData?.trademarkRegistration ?? '',
-          isSix: false,
-          isSeven: true,
-          isThree: false,
-          point1: variables.staticData?.idealForIndividualProprietor ?? '',
-          point2: variables.staticData?.filingOfTrademarkApplication ?? '',
-          point3: variables.staticData?.drafingOfTrademarkApplicationForm ?? '',
-          point4:
-              variables.staticData?.adviceOnTrademarkClassAndClassification ??
-                  '',
-          point5: variables.staticData?.nameSearchAndApproval ?? '',
-          point6: variables.staticData?.regularUpdatesUntilRegistration ?? '',
-          point7:
-              variables.staticData?.excludesTheReplyToTrademarkObjections ?? '',
+        InkWell(
+          onTap: () {
+            setState(() {
+              isRequirements = !isRequirements;
+            });
+          },
+          child: Padding(
+            padding: kLeftRightPadding50,
+            child: Row(
+              crossAxisAlignment: CrossAxisAlignment.center,
+              children: [
+                Flexible(
+                  child: Text(
+                    capitalizeFirst(
+                      widget.variables.staticData
+                              ?.trademarkRegistrationRequirementsInTheUae
+                              ?.capitalizeFirstLetter() ??
+                          '',
+                    ),
+                    style: context.titleMedium
+                        ?.copyWith(color: allColors.lightTextColor),
+                    textAlign: TextAlign.center,
+                  ),
+                ),
+                Icon(
+                  isRequirements
+                      ? Icons.keyboard_arrow_up
+                      : Icons.keyboard_arrow_down,
+                  color: context.onSurfaceColor,
+                  size: 25,
+                ),
+                SizedBox(
+                  width: 20.w,
+                )
+              ],
+            ),
+          ),
         ),
-        ExpandableCards(
-          price: variables.staticData?.aed_1135_5Vat ?? '',
-          name: variables.staticData?.trademarkSearch_Price ?? '',
-          isSix: false,
-          isSeven: false,
-          isThree: true,
-          point1: variables
-                  .staticData?.enablesTheCustomerToSubmitARequestForSearc ??
-              '',
-          point2: variables
-                  .staticData?.facilitatesTheInvestigationOfExistingOrRegis ??
-              '',
-          point3: variables
-                  .staticData?.assessesTheSimilarityBetweenTheProposedTrade ??
-              '',
-          point4: '',
-          point5: '',
-          point6: '',
-          point7: '',
+        if (isRequirements) ...[
+          ExpandableCards(
```

Comment:
we have to create separate widget for every screen expandable widget.

check there is too much logic. it's hard to understand the code